radio/src/targets/horus/board.cpp

```diff
@@ -236,11 +236,13 @@ void boardInit()
   sportUpdateInit();
 #endif
 
-  ledBlue();
+  ledRed();
```

Comment:
Since we're using this for RTC diagnostic, perhaps better to put it within the RTC block just below? And what about doing the same for the other targets?

Reply:
The whole point of this exercise is to know when the RTC operation finishes, which will tell us if reading of RTC data was successful. If we stored that in RTC we wouldn't have known that it's broken.

Reply:
Don't think that is what I said... or did I?

```
#if defined(RTCLOCK) && !defined(COPROCESSOR)
  ledRed();
  rtcInit(); // RTC must be initialized before rambackupRestore() is called
#endif
```

btw, `&& !defined(COPROCESSOR)` can go also, as this is a legacy condition that is no longer used.

Reply:
Done.